kibana-config/mapping.json

```diff
@@ -1,3 +1,3 @@
 [
-"{\".kibana\":{\"mappings\":{\"config\":{\"properties\":{\"buildNum\":{\"type\":\"long\"},\"defaultIndex\":{\"type\":\"string\"}}},\"index-pattern\":{\"properties\":{\"fieldFormatMap\":{\"type\":\"string\"},\"fields\":{\"type\":\"string\"},\"intervalName\":{\"type\":\"string\"},\"timeFieldName\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"}}},\"dashboard\":{\"properties\":{\"description\":{\"type\":\"string\"},\"hits\":{\"type\":\"integer\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"panelsJSON\":{\"type\":\"string\"},\"timeFrom\":{\"type\":\"string\"},\"timeRestore\":{\"type\":\"boolean\"},\"timeTo\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"}}},\"visualization\":{\"properties\":{\"description\":{\"type\":\"string\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"savedSearchId\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"},\"visState\":{\"type\":\"string\"}}},\"search\":{\"properties\":{\"columns\":{\"type\":\"string\"},\"description\":{\"type\":\"string\"},\"hits\":{\"type\":\"integer\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"sort\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"}}}}}}"
+"{\".kibana\":{\"mappings\":{\"dashboard\":{\"properties\":{\"description\":{\"type\":\"string\"},\"hits\":{\"type\":\"integer\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"panelsJSON\":{\"type\":\"string\"},\"timeFrom\":{\"type\":\"date\",\"format\":\"strict_date_optional_time||epoch_millis\"},\"timeRestore\":{\"type\":\"boolean\"},\"timeTo\":{\"type\":\"date\",\"format\":\"strict_date_optional_time||epoch_millis\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"}}},\"config\":{\"properties\":{\"buildNum\":{\"type\":\"long\"},\"defaultIndex\":{\"type\":\"string\"},\"embed:enableFilters\":{\"type\":\"boolean\"},\"embed:showTitle\":{\"type\":\"boolean\"}}},\"search\":{\"properties\":{\"columns\":{\"type\":\"string\"},\"description\":{\"type\":\"string\"},\"hits\":{\"type\":\"integer\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"sort\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"}}},\"visualization\":{\"properties\":{\"description\":{\"type\":\"string\"},\"kibanaSavedObjectMeta\":{\"properties\":{\"searchSourceJSON\":{\"type\":\"string\"}}},\"savedSearchId\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"},\"version\":{\"type\":\"integer\"},\"visState\":{\"type\":\"string\"}}},\"index-pattern\":{\"properties\":{\"fields\":{\"type\":\"string\"},\"timeFieldName\":{\"type\":\"string\"},\"title\":{\"type\":\"string\"}}}}}}"

```

Comment:
Goes with data.json, two new fields have been added in kibana settings and are now defined in this data.json file